### DISCUSSION TITLE
display touch

### DISCUSSION THREAD

**Discussion by paolopa65** [Jyers/Marlin#819] [Q&A] [answered]
It would be possible to mount this Creality Touch display on Firmwaare v1.3.2?
If possible what changes do you need?
It would be enough to load the Touch display firmware on the Creality website?
![Cattura](https://user-images.githubusercontent.com/79114433/118187159-e2bde580-b43e-11eb-9c62-18063e03e1ab.JPG)

**Comment by Jyers** [comment, marked as answer]
There are no immediate plans to support this display. I do encourage you to take a look here though https://github.com/CR6Community/Marlin as this firmware is for that display
